backend/builder/image_builder/image_builder.py

```diff
@@ -284,6 +295,10 @@ def _build_container(dockerfile_mount_path: str, image_tag: str) -> kubernetes.c
         )
         volume_mounts.append(docker_config)
 
+    if PRIVATE_CA_ENABLED:
+        docker_config = kubernetes.client.V1VolumeMount(name=CA_SECRET_NAME, mount_path="/kaniko/ssl/certs")
```

Comment:
@guilhem-barthes how the system knows that it must calls the file `ca-certificates.crt` ?

Reply:
That's the default for ssh configuration